core/arch/arm/kernel/proc_a32.S

```diff
@@ -89,6 +89,15 @@ UNWIND(	.fnstart)
 UNWIND(	.fnend)
 END_FUNC cpu_spin_unlock
 
+/* int cpu_mmu_enabled(void) - check MMU enabled or not */
+FUNC cpu_mmu_enabled , :
```

Comment:
Put it in `core/arch/arm/mm/core_mmu.c`, it will be useful for other platforms using non-linear mapping of devices.